_data/blog_posts.yml

```diff
@@ -1,3 +1,12 @@
+- title: "A talk on the new AWS Application Load Balancer, updates to ECS, and Kinesis Analytics"
+  url: "https://blog.gruntwork.io/a-talk-on-the-new-aws-application-load-balancer-updates-to-ecs-and-kinesis-analytics-abb599cb3cb8#.vhbek5ce2"
+  thumbnail_url: "https://i.imgur.com/KkniqPD.png"
```

Comment:
Ugh, Medium takes a single image and places it in 3 different proportions, cutting it off at arbitrary points, depending on where you're viewing the article or headline. I added something that's workable but doesn't look that great.

I'm going to merge this so we have it live and I can move on, but feel free to tinker with this if you have other ideas. Ultimately, I had to format my image to match the proportions in the pages that list multiple articles, but it still doesn't look very good overall.